### PR TITLE
site: Make sure is_atomic is defined

### PR DESCRIPTION
**Pull request by badone**
configure_firewall tests the is_atomic variable if the firewalld package
is not present.

Signed-off-by: Brad Hubbard <bhubbard@redhat.com>

### CODE REVIEW

**Comment by guits** [comment]
`is_atomic` is already defined in `ceph-facts`, simply including it where @leseb pointed out is enough although it's a pitty to have to play the whole role just for this...

**Comment by badone** [comment]
Done. Tested with stable-3.2 as well and the equivalent change works. Thanks @leseb and @guits

File: site.yml.sample [hunk 1]
```diff
@@ -76,6 +76,8 @@
         name: ceph-defaults
     - import_role:
         name: ceph-validate
+    - import_role:
```
Comment (leseb) [inline site.yml.sample]: Nit, usually we prefer to call `ceph-facts` right after `ceph-defaults`. Thanks.

Comment (badone) [reply site.yml.sample]: OK, let me reorder that :)

**Comment by badone** [comment]
@leseb I think this is good now?

**Comment by leseb** [comment]
@badone yes, thanks!

**Pull request merged**